build/nuget/uno.winui.common.targets

```
@@ -11,12 +11,17 @@
 	<!-- Feature configuration -->
 	<PropertyGroup>
 		<UnoDragDropExternalSupport Condition="'$(UnoDragDropExternalSupport)'==''">true</UnoDragDropExternalSupport>
+		<UnoEnableDynamicDataTemplateUpdate Condition="'$(UnoEnableDynamicDataTemplateUpdate)'==''">false</UnoEnableDynamicDataTemplateUpdate>
```

```suggestion
		<UnoEnableDynamicDataTemplateUpdate Condition="'$(UnoEnableDynamicDataTemplateUpdate)'==''">true</UnoEnableDynamicDataTemplateUpdate>
```
BUT: If $(Optimize) is "true", it should be false instead...